ShopSpiders/ShopSpiders/spiders/adidas.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+
+
+class AdidasSpider(scrapy.Spider):
+    adidas_products = ["calzado-hombre","ropa-hombre","accesorios-hombre","performance-hombre"
+    ,"calzado-mujer","ropa-mujer","accesorios-mujer","performance-mujer"
+    ,"ninos?prefn1=age&prefv1=8-16%20A%C3%B1os&srule=newest-to-oldest"
```

Comment:
This and below line crawl the same items(same items but different order). Also we need to add a way to delete duplicates.

Reply:
Also, there's no other way to crawl all items and this hardcoded way? maybe using just https://m.adidas.pe/on/demandware.store/Sites-adidas-PE-Site/es_PE/Search-BrowseCatalog?fid=%22%2Fcopa-mundial-sitemap%2Ehtml this page that contains all items.